www/%username/widgets/%type.spt

```diff
@@ -35,10 +36,12 @@ response.headers[b'Vary'] = b'Accept-Language'
     ><span style="display: block; padding: 0 15px;"
         ><span style="color: #f6c915; position: absolute; left: -2px;">&#10132;</span
         >{% if participant.kind == 'individual' %}{{
-            _("I donate {0}{1}{2}", A, participant.giving, B) if t[0] == 'g' else
-            _("I receive {0}{1}{2}", A, participant.receiving, B)
+            _("I donate {0}{1}{2}", span_open, participant.giving, span_close) if t == 'giving.js' else
+                (_("I receive {0}{1}{2}{3} per week,{3} my goal is {0}{4}{2}", span_open, participant.receiving, span_close, br, goal) if goal else
+                 _("I receive {0}{1}{2}", span_open, participant.receiving, span_close))
         }}{% else %}{{
-            _("We donate {0}{1}{2}", A, participant.giving, B) if t[0] == 'g' else
-            _("We receive {0}{1}{2}", A, participant.receiving, B)
+            _("We donate {0}{1}{2}", span_open, participant.giving, span_close) if t == 'giving.js' else
+            (_("We receive {0}{1}{2}{3} per week,{3} my goal is {0}{4}{2}", span_open, participant.receiving, span_close, br, goal) if goal else
+             _("We receive {0}{1}{2}", span_open, participant.receiving, span_close))
```

Comment:
- wrapping the `else` branches in parentheses is unnecessary
- `if goal` is not enough, it needs to be `if goal and goal.amount > 0`
- `{0}{1}{2}{3}` is a lot of abstract placeholders, translators will wonder what they're for
- if we're using "per week" in one widget we should add it to all the others as well

Reply:
> - `if goal` is not enough, it needs to be `if goal and goal.amount > 0`

On second thought `if goal and goal > receiving` may be more appropriate.